docs/deployment/docker-compose-weave.md

```diff
@@ -64,6 +64,7 @@ This will send some traffic to the application, which will form the connection g
 <!-- deploy-doc-hidden run-tests
 
     docker run -td -\-name healthcheck andrius/alpine-ruby /bin/sh
```

Comment:
1. It's safer to build all things in the `pre-install` phase. Could you move all things that do not involve starting the test 
there?
2. What about introducing a Dockerfile that is turned into a test container? Then it can be published to the Hub and shared by all `docker` tests.